index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="description" content="A mobile website on a personal portfolio">
+  <meta name="author" content="Promise Chukwu">
+  <meta name="keywords" content="website,portfolio,Promise Chukwu,project">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="index.css"/>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+</head>
+<body>
+  <article class="headline">
+  <header>
+    <p class="logo">My Logo</p>
+    <img src="icons/Union.png" id="menu-icon" alt="menu-icon"/>
+  </header>
+  <section class="information">
+    <section class="text">
+      <h1>I'm Promise.<br/>Glad to see<br/>you!</h1>
+      <p id="primary-text">I'm a software developer! I can help you build a product, feature or website look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p> 
+    </section>
+    <footer>
+      <h2 id="pm">LET'S CONNECT</h2>
+     <div class="social-media-icons">
+      <a href="https://x.com/@chukwus618"><img src="icons/x.svg" alt="x-handle"></a> 
+      <a href="https://linkedin.com/in/smart-chukwu-89b7072b0/"><img src="icons/Linkedin.svg" alt="Linkedin-handle"></a>
+      <a href="https://medium.com/@chukwus618"><img src="icons/median.svg" alt="Median-handle"></a>
+      <a href="https://github.com/chukwus618"><img src="icons/github.svg" alt="github-handle"></a>
+      <a href="https://wellfound.com/u/smart-chukwu"><img src="icons/wellfound.svg" alt="Wellfound-handle"></a>
+     </div>
```

Comment:
[OPTIONAL CHANGES]
I'm sure that you are suspecting this already 😆 and yes, writing semantic html tags is a great deal. 
- [ ] Did you notice that this looks like a list of social media account? Yeah, an `ul` or a `menu` tag would have handled this well with some `li` accompanying the tag, which will not only make your code SEO-friendly but also make the styling (CSS Selection) very easy without repetition of attributes or even classes.